core/backend/src/BriefcaseManager.ts

```diff
@@ -345,6 +345,7 @@ export class BriefcaseManager {
     const files = IModelJsFs.readdirSync(folderPathname);
     for (const file of files) {
       const curPath = path.join(folderPathname, file);
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       const locStatus = (IModelJsFs.lstatSync(curPath)!.isDirectory) ? BriefcaseManager.deleteFolderAndContents(curPath) : BriefcaseManager.deleteFile(curPath);
```

Comment:
this can be "?".

Reply:
@DanielToby this all looks good, thank you!

(but FYI the etiquette is to let the person who opened the conversation resolve it)

Reply:
@kabentley Ah, my mistake. Thanks for the heads up!